mlc_llm/relax_model/param_manager.py

```diff
@@ -685,30 +694,13 @@ def _dequantize(
         param : Parameter
             The parameter whose quantized tensors are to be dequantized.
 
-        quantized_tuple : relax.Var
-            The relax.Var of the quantized tensors of all parameters in the model.
-
-        bb : relax.BlockBuilder
-            The Relax BlockBuilder used for inserting the dequantization computations.
-
-        func_name : str
-            The name of the  function which dequantization is applied to.
-
         qparams : List[relax.Var]
-            The quantized parts of the parameter.
-            By default it is `None`, in which case we will get the quantized parts
-            from `quantized_tuple`.
+            The relax.Var of the quantized tensors of all parameters in the model.
 
         Returns
         -------
         The dequantized parameter, in the form of a relax.Var.
         """
-        if not qparams:
-            # Get the corresponding Relax vars of the quantized tensors of this parameter.
-            qparams: List[relax.Var] = []
-            for qparam_idx in self.param2qrange[param]:
-                qparams.append(bb.emit(relax.TupleGetItem(quantized_tuple, qparam_idx)))
-
```

Comment:
Just to confirm, do we still need to check the case where `qparams` is None?

Reply:
We don't, no.  Previously, the calling scope could either provide a `quantized_tuple: relax.Var` variable, from which `qparams` would be determined, or could provide a `qparams: List[relax.Var]` directly, in which case `quantized_tuple` would be ignored entirely.  This commit now requires the calling scope to provide `qparams` in all cases, so this check is no longer required.

This function is only called from two locations, and so the calling site that previously used the `quantized_tuple` argument is also updated by this PR.